packages/input-iban/README.md

```diff
@@ -90,3 +90,25 @@ export const countryRestrictions = () => html`
   <small>Demo instructions: you can use NL20 INGB 0001 2345 67</small>
 `;
 ```
+
+### Blacklisted Country
+
+By default, we validate the input to ensure the IBAN is valid.
+To get the default feedback message for this default validator, use `loadDefaultFeedbackMessages` from `@lion/form-core`.
+
+In the example below, we show how to use an additional validator that blocks IBANs from certain countries.
+
+```js preview-story
+export const blacklistedCountry = () => html`
+  <lion-input-iban
+    .modelValue=${'RO89RZBR6997372848645577'}
+    .validators=${([new IsNotCountryIBAN('RO')], [new IsNotCountryIBAN('ES')])}
```

Comment:
Isn't this a bit against how validators work atm?

The system was designed to have one instance of a certain validator per FormControl.
per validator type we can check whether it's either valid or invalid for this Validator.
The 2nd instance in the example above will probably cancel out the first one, leading to:
`validationStates.error.IsNotCountryIban => false`

(=> action point, check for this during setup and when new Validators are added)

Maybe we can do smth like `new isNonBlackListedIban(['RO', 'ES'])` ?

Reply:
I would be down with keeping the current name (IsNotCountryIBAN) and passing in an array, but this significantly complicates the translation message which is why I was okay with having multiple instances instead

Reply:
Yeah, I understand. For that to work we would need to allow for the validator to return metadata instead of just a Boolean, maybe.
About the name, it was not completely intuitive to me personallt, but it best aligns with the current `IsCountryIBAN `
indeed.

The validationStates after a better check would actually be marked as error, because it only sets it to true for everything that ends up being invalid (just one instance of `IsNotCountryIBAN`).

Alright, I can approve it then :)